### PR TITLE
Add transaction (a list of deltas) as a primitive for applying sets o…

### PR DESCRIPTION
**Pull request by barakmich**
…f changes

This will also line up with removing "AddQuadSet" and help with some more intricate types of updates. It's not full-query-capability yet, but we'll need the primitive to do that anyway.

### CODE REVIEW

File: graph/transaction.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+import (
+	"github.com/google/cayley/quad"
+)
+
+type Transaction struct {
+	Deltas []Delta
+}
+
+func CreateTransaction() *Transaction {
```
Comment (jzelinskie) [inline graph/transaction.go]: `NewTransaction` would be more idiomatic

File: imports.go [hunk 2]
```diff
@@ -18,6 +18,7 @@ var StartMorphism = path.StartMorphism
 var StartPath = path.StartPath
 
 var RawNext = graph.Next
+var CreateTransaction = graph.CreateTransaction
```
Comment (jzelinskie) [inline imports.go]: move these exports into a single `var` block

**Comment by jzelinskie** [comment]
LGTM

**Comment by Quentin-M** [comment]
LGTM

**Pull request merged**